fs/fs.go

```diff
@@ -235,6 +235,7 @@ func (cfs *FS) WriteFile(name string, src fs.File) error {
 
 // Remove deletes a file from the Charm Cloud server.
 func (cfs *FS) Remove(name string) error {
+	name = strings.TrimPrefix(name, "charm:")
```

Comment:
I wonder if this is the right place for such a check. Maybe this should be the responsibility of the caller?

Reply:
Note: this is a generic question, really. Your fix is in line with the rest of the code, e.g. `Open`.

Reply:
yeah, I'm not sure as well... followed the same idea of the other methods, like you noted, `open`, for example...

I do see quite a bit of places repeating the `charm:` check though...

Reply:
We could move it into `EncryptPath` since that's what's called after stripping the prefix everywhere: https://github.com/charmbracelet/charm/blob/main/fs/fs.go#L358 Basically it would prep a "Charm compatible" path.